core_backend/tests/api/conftest.py

```diff
@@ -308,8 +329,8 @@ async def mock_dashboard_stats(*arg: str, **kwargs: str) -> QuestionDashBoard:
     statistics.
     """
     return QuestionDashBoard(
-        six_months_question=[random.randint(0, 100) for _ in range(6)],
-        sis_months_upvote=[random.randint(0, 100) for _ in range(6)],
+        six_months_questions=[random.randint(0, 100) for _ in range(6)],
```

Comment:
How in the world did tests pass with this error? We may need to check the tests for Dashboards
Also my IDE threw an error

Reply:
Ah! looks like it was not mocking correctly and so was calling the actual endpoint. This was returning zeroes for both values and the test was passing.

@lickem22 - Can i suggest:
1. Move `mock_dashboard_stats` to `test_question_dashboard.py`. It doesnt need to be in conftest
2. It should take two arguments: questions, upvotes
3. Use functools.partial to create a function with these argument set to (0, 0), (1000, 1000), (1000, 0), (0, 1000) and anything else you think makes sense
4. mock `core_backend.app.question_dashboard.routers.get_dashboard_stats` with this partial
5. Check that returned values match the values set in #3

Reply:
Hi Sid, I am not sure I understand 5. What values are set in the PR you linked ?